### PR TITLE
Cypress setup

### PR DESCRIPTION
**Pull request by aleciabenjamin**
#### Description

* Install Cypress
* Remove example specs

#### What I have learned working on this feature: 
I learned how simple it is to install cypress as well as about intelligent code completion.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,15 +11,22 @@
...
+		"cy:open": "yarn start --silent & cypress open"
   },
   "eslintConfig": {
-    "extends": "react-app"
-  },
+		"extends": "react-app",
+		"globals": {
+			"cy": "cy"
+		}
+	},
```
Comment (tochman) [inline package.json:24]: Code format is off.

Comment (aleciabenjamin) [reply package.json:24]: Strange.  Looks good in vsc.  Will check it out...

Comment (aleciabenjamin) [reply package.json:24]: Not sure why it looks like that here, but in vsc it looks good.

Comment (tochman) [reply package.json:24]: Yes, that IS weird. It’s fixable though...

**Comment by nevroje** [comment]
Shouldn't the package.json file already be formatted/readable when u generate it?

File: package.json [hunk 2]
```diff
@@ -1,25 +1,32 @@
...
+		"test": "react-scripts test",
+		"eject": "react-scripts eject",
+		"cy:open": "yarn start --silent & cypress open"
+	},
+	"eslintConfig": {
+		"extends": "react-app",
+		"globals": {
+			"cy": "cy"
+		}
+	},
```
Comment (Joberg95) [inline package.json:24]: What happened with the formatting? isnt it best to just not touch this file ?

Comment (aleciabenjamin) [reply package.json:24]: I changed the eslintConfig and scripts.

**Review by yanqiuzhang — APPROVED** [review]

**Review by NiciP — APPROVED** [review]

**Review by Joberg95 — APPROVED** [review]

**Pull request merged**